lab_1/start.py

```diff
@@ -15,11 +16,30 @@
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'de.txt'), 'r', encoding='utf-8') as file_to_read:
         de_text = file_to_read.read()
 
+    with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'la.txt'), 'r', encoding='utf-8') as file_to_read:
+        la_text = file_to_read.read()
+
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'unknown.txt'), 'r', encoding='utf-8') as \
             file_to_read:
         unknown_text = file_to_read.read()
 
+    lang_unk = main.create_language_profile('unk', unknown_text, [])
+    lang_en = main.create_language_profile('en', en_text, [])
+    lang_de = main.create_language_profile('de', de_text, [])
+    lang_la = main.create_language_profile('la', la_text, [])
+
+    TOP_N = 4
+
+    langs = [lang_en, lang_de, lang_la]
+
+    det = main.detect_language(lang_unk, lang_en, lang_de, TOP_N)
+
+    det_adv = main.detect_language_advanced(lang_unk, langs, [lang_la, lang_de], TOP_N)
```

Comment:
can you explain in writing what happens in `start.py`?

Reply:
First of all, program opens .txt files whith the texts, then I created variables whith the language profiles of all languages (en, de, la and unknown language) (using create_language_profile function from main.py) and TOP_N variable with the number of top used words I should use (in my case it is 4). Then goes list with all profiles except for the unknown language. In the end there is RESULT where detect_language_advanced function from main.py is used (where I actually detect language) and print where the result can be seen. Also there is a check whether my program is working properly that compares expected result and the result my function got.